spec/serverSpec.js

```diff
@@ -0,0 +1,30 @@
+/* You'll need to have the express server running
+ * for these tests to pass. */
+
+const { expect } = require('chai');
+const request = require('request');
+
+describe('the server', () => {
+  it('get all the reviews when sending a /reviews GET request', (done) => {
```

Comment:
I think this may become a couple tests, actually:
* test the seeding script to see if it generates 10k records (perhaps running it against a test collection)
* test the endpoint to see what it returns as a response (using mocks / stubs)